src/nunjucks/moduk/components/breadcrumbs/__tests__/macro.test.ts

```diff
@@ -0,0 +1,36 @@
+import { describe, expect, it } from 'vitest'
+
+import { render } from '../../../../../test-utils'
+
+describe('Breadcrumbs', async () => {
+  it('adds the moduk CSS class and renders the content', () => {
+    const items = [
+      {
+        text: 'Home',
+        href: '#',
+      },
+      {
+        text: 'Passports, travel and living abroad',
+        href: '#',
+      },
+      {
+        text: 'Travel abroad',
+        href: '#',
+      },
+    ]
+
+    const element = render(`
+{%- from "moduk/components/breadcrumbs/macro.njk" import modukBreadcrumbs -%}
+
+{{ modukBreadcrumbs({
+  items: ${JSON.stringify(items)}
+}) }}
+`)
+
+    expect(element).toHaveClass('moduk')
+
+    element.querySelectorAll('li a').forEach((a, i) => {
```

Comment:
Would be good to use more descriptive names for the arrow function arguments:

```suggestion
    element.querySelectorAll('li a').forEach((element, index) => {
```

(Or `anchor` instead of `element`.)

Reply:
Also would be good to try to use the Testing Library functions e.g. `getAllByRole(element, 'link')`.

But looks like that'll need  `render()` changed to return an `HTMLElement` – the if statement in it could be changed to `if (!(document.body.firstElementChild instanceof HTMLElement)) {` to make that work.

Another option is to check the text content in an end-to-end test instead of here.